other/require-opa-env/require-opa-enabled.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: validate-env-vars
+  annotations:
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/title: Check envirnoment variables
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/subject: Pod
+    kyverno.io/kubernetes-version: "1.22-1.23"
```

Comment:
Can you also test on K8s 1.24 and/or 1.25? Update as needed.

Reply:
I am happy to test with kyverno-cli, but I am not sure how to test it with Kubernetes.

Reply:
KinD? K3d? Minikube?